src/main/java/solitaire/game/CardStack.java

```diff
@@ -67,8 +69,65 @@ private void solveFlipType(int flipType) {
                 for (int i = 0; i < cards.size(); i++) {
                     cards.get(i).setFaceDir(true);
                 }
+                break;
+            case FLIPTYPE_NONE:
+                for (int i = 0; i < cards.size(); i++) {
+                    cards.get(i).setFaceDir(false);
+                }
+                break;
+        }
+    }
+
+    private void calcBounds(int x, int y) {
+        int width = (this.tieredXOffset * (Math.max(0, this.getCardCount() - 1))) + ImageLoader.cardTexWidth;
+        int height = (this.tieredYOffset * (Math.max(0, this.getCardCount() - 1))) + ImageLoader.cardTexHeight;
+
+        this.bounds = new Rectangle(x, y, width, height);
+    }
+
+    public boolean inBounds(int x, int y) {
+        boolean b = false;
+        try{
+            b = bounds.contains(x, y);
+        }catch(NullPointerException e){}
```

Comment:
This is bad practice, if `NullPointerException`s are being thrown we should look into and remedy the cause